source/default_mode/SymWorld.h

```diff
@@ -650,7 +650,6 @@ class SymWorld : public emp::World<Organism>{
     int numupdates = my_config->UPDATES();
     for (int i = 0; i < numupdates; i++) {
       if(verbose && (i%my_config->DATA_INT())==0) {
```

Comment:
I didn't mean for you to delete this print statement as well, just the one that you had put in